### ISSUE TITLE
No overlays. I tried 6.1, 6.0, and 5.1

### ISSUE THREAD

**Issue by zkiraly** [kubilus1/autoortho#408]
**Describe the bug**
Autoortho does not generate an overlay folder in the custom scenery folder, and does not place overlay folder in the ini file either. Here is a screenshot of the folder.

**Screenshots**
![Custom_scenery](https://github.com/kubilus1/autoortho/assets/57362/27ee1d60-a82a-4fd6-9de7-7b3a5d05491b)

**(please complete the following information):**
 - OS: Windows 10
 - XPlane Version:XP12.07
 - AutoOrtho Version 6.1, 6.0, 5.1
 - Filesystem type NTFS

**Comment by jonaseberle** [comment]
I don't know if there is a problem with the pack installer but as a workaround you can download the overlays from here directly:
https://github.com/kubilus1/autoortho-scenery/releases

**Comment by Jeff2122** [comment]
I tested right quick to see if there is an issue with the server or install process. It worked great. I didn't need to copy anything. 

I keep my scenery on a remote drive normally but for this exercise I deleted all my symlinks and installed Australia on my main XP drive, in Custom Scenery as a standard installation. Using v0.5.1-3. 

It should work fine as long as you 1. Run AO executable as administrator. and 2. You specify your target folder correctly i.e. "z:\X-Plane 12\Custom Scenery" not just "z:\X-Plane 12". 

Noticed that when I browsed for the folder, I had to be careful to select the right folder, as it doesn't act like a native Windows select, it grabs whatever is under the mouse so I mouse to Custom Scenery and drag off, if that makes sense. Maybe that's what happened to you. BTW I used the installer in v0.7.0 a few days ago to iinstall the data for Australia on my remote drive and it worked fine too.


    OS: Windows 10
    XPlane Version:XP12.07
    AutoOrtho Version 0.5.1.-3, 0.6.0, 0.6.1, 0.7.0
    Filesystem type NTFS

DL speed ws 17MB/s here
